### DISCUSSION TITLE
[Alfajores] Blockscout UI lagging several minutes

### DISCUSSION THREAD

**Discussion by kathaypacific** [celo-org/blockscout#920] [Q&A] [answered]
we've had some end-to-end test failures that we believe are stemming from blockscout running behind by several minutes on Alfajores. I'm seeing on block explorer that the latest block is lagging by a few minutes (though it varies) https://explorer.celo.org/alfajores/blocks and a transfer I made on Alfajores 10 minutes ago is not found-- it has ID 0x0a57dc6b4943564f72cd1493324b461e0b9b802d919c630de3c2f2501de5bae1 . (Interestingly, it is showing up on a preview when I paste it into the search bar, but not when I hit return.)

![Screenshot 2023-06-19 at 3 45 15 PM](https://github.com/celo-org/blockscout/assets/20150449/447ad38a-52e2-477c-be87-28e9f805a1dc)

![Screenshot 2023-06-19 at 3 44 40 PM](https://github.com/celo-org/blockscout/assets/20150449/c340e09b-65e3-4f82-ab15-a6282ba410d8)

**Comment by jcortejoso** [comment, marked as answer]
Hi @kathaypacific.
We detected some flooding activity on Alfajores, and took some decisions to limit the impact.
![image](https://github.com/celo-org/blockscout/assets/5635989/a7c40d09-861c-4e8a-a2d7-2efa816ef4e5)

Unfortunately this impacted on Blockscout Alfajores performance with the DB, and it  had to recover all these "pending" tx data:
![image](https://github.com/celo-org/blockscout/assets/5635989/b0e84e8a-9638-4ccf-934d-9449a97e0b3c)

There are yet some pending indexing operations, but I hope this can be normalized in the next hours.
